components/legacy_sidebar/header/sidebar_header_dropdown_button.jsx

```diff
@@ -72,7 +72,13 @@ export default class SidebarHeaderDropdownButton extends React.PureComponent {
                         >
                             {'@' + this.props.currentUser.username}
                         </div>
-                        <CustomStatusEmoji/>
+                        <CustomStatusEmoji
+                            showTooltip={true}
+                            emojiStyle={{
```

Comment:
CustomStatusEmoji has a prop called size, so technically the height should be dependent on the size.

Reply:
We need different styles in different places. Height property is not needed in the dropdown.

Being able to customize the span css can be useful for the rest of the components as well.